MAUI/DateTime-Range-Selector/tooltip.md

```diff
@@ -59,11 +59,60 @@ rangeSelector.Content = chart;
 
 ![RangeSelector tooltip](images/tooltip/tooltip.png)
 
-N> Refer [here](https://help.syncfusion.com/maui/range-selector/events) to customize the tooltip text format through the Range Selector events.
+## Show always
+
+Always shows a tooltip with and without the thumb interaction by setting the [`SliderTooltip.ShowAlways`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Sliders.SliderTooltip.html#Syncfusion_Maui_Sliders_SliderTooltip_ShowAlways) property. The default value of the [`SliderTooltip.ShowAlways`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Sliders.SliderTooltip.html#Syncfusion_Maui_Sliders_SliderTooltip_ShowAlways) property is `False`.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<ContentPage 
+             ...
+             xmlns:sliders="clr-namespace:Syncfusion.Maui.Sliders;assembly=Syncfusion.Maui.Sliders"
+             xmlns:charts="clr-namespace:Syncfusion.Maui.Charts;assembly=Syncfusion.Maui.Charts">
+    
+    <sliders:SfDateTimeRangeSelector Minimum="2010-01-01" 
+                                     Maximum="2018-01-01" 
+                                     RangeStart="2012-01-01" 
+                                     RangeEnd="2016-01-01">
+         
+         <sliders:SfDateTimeRangeSelector.Tooltip>
+            <sliders:SliderTooltip ShowAlways="True"/>
+         </sliders:SfDateTimeRangeSelector.Tooltip>
+     
+     <charts:SfCartesianChart>
+         ...
+     </charts:SfCartesianChart>
+    
+    </sliders:SfDateTimeRangeSelector>
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfDateTimeRangeSelector rangeSelector = new SfDateTimeRangeSelector();
+rangeSelector.Minimum = new DateTime(2010, 01, 01);
+rangeSelector.Maximum = new DateTime(2018, 01, 01);
+rangeSelector.RangeStart = new DateTime(2012, 01, 01);
+rangeSelector.RangeEnd = new DateTime(2016, 01, 01);
+rangeSelector.Tooltip = new SliderTooltip();
+rangeSelector.Tooltip.ShowAlways = true;
+SfCartesianChart chart = new SfCartesianChart();
+rangeSelector.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![RangeSelector show always tooltip](images/tooltip/show-always-tooltip.gif)
+
+N> Refer [here](https://help.syncfusion.com/maui/datetime-range-selector/events-and-commands#tooltip-text-format) to customize the tooltip text format through the DateTime Range Selector events.
```

Comment:
Content is wrong. Update it as per discussed

Reply:
Reupdated the content as dicussed.